### ISSUE TITLE
Broken links in LICENCE.md

### ISSUE THREAD

**Issue by slafayIGN** [IGNF/validator#171]
The links to www.cecill.info seems to be broken.

The links work without www but the licence text has a problem with special characters in french : http://cecill.info/licences/Licence_CeCILL-B_V1-fr.txt

I suggest either : 

* use the same link as referenced here : https://www.data.gouv.fr/fr/licences (links to https://spdx.org/licenses/CECILL-B.html#licenseText in english only)
* copy the full licence text in LICENCE.md
* use another compatible licence

**Comment by mborne** [comment]
Thanks! Same problem here : https://github.com/IGNF/geoportal-extensions/blob/develop/LICENCE.md, https://github.com/IGNF/ol-geometry-editor/blob/master/LICENCE.md, ...

What about removing "www." and notifing cecill.info contact about the LATIN1 charset issue?

**Comment by slafayIGN** [comment]
I just emailed webmaster@cecill.info and technique@cecill.info about both charset and a possible redirection from www issues. I mentionned the second one because we're maybe not the only ones referencing a broken link.

Wikipedia changed the links back in february 2018 : https://fr.wikipedia.org/w/index.php?title=Licence_CeCILL&diff=145872461&oldid=128580339
And it appears there is also a certificate issue... so we shouldn't use https.

This is really an old website and CeCILL is not anymore in the official recommandations of the french government (as of 2018) : https://disic.github.io/politique-de-contribution-open-source/
